shunting-yard-replicator/src/test/java/com/hotels/shunting/yard/replicator/exec/external/MarshallerTest.java

```diff
@@ -73,6 +85,54 @@ public void typical() throws Exception {
         .contains("    database-name: databaseName")
         .contains("    table-name: tableName")
         .contains("    table-location: replicaTableLocation");
+
+  }
+
+  @Test
+  public void typicalWithGraphite() throws Exception {
+    CircusTrainConfig config = CircusTrainConfig
+        .builder()
+        .sourceName("sourceName")
+        .sourceMetaStoreUri("sourceMetaStoreUri")
+        .replicaName("replicaName")
+        .replicaMetaStoreUri("replicaMetaStoreUri")
+        .copierOption("p1", "val1")
+        .copierOption("p2", "val2")
+        .replication(ReplicationMode.FULL, "databaseName", "tableName", "replicaTableLocation", Arrays.asList("part"),
+            Arrays.asList(Arrays.asList("partval")))
+        .graphite(graphiteConfig)
+        .build();
+
+    File file = tmp.newFile("conif.yml");
```

Comment:
Not that it matters, and I know that this is from the other tests, but isn't there a typo in the name of the file?

Also, since this in all the tests, it could be put at the top, where `graphiteConfig` is initialised.